builtin/mainmenu/settings/components.lua

```diff
@@ -368,6 +368,12 @@ local function noise_params(setting)
 		setting = setting,
 
 		get_formspec = function(self, avail_w)
+			-- The "defaults" noise parameter flag doesn't reset a noise
```

Comment:
I might be missing something, where is this "defaults" parameter?

Reply:
Noise param settings can have any combination of these three flags set: `defaults`, `eased` and `absvalue`. `eased` and `absvalue` have an actual meaning, but `defaults` doesn't do a lot except also enabling `eased`. (That's what I think, at least, I haven't looked into this very deeply.)

Because of the way the flags are presented in the dialog, one could think that enabling `defaults` resets a noise param setting to its default value. To avoid this misunderstanding, I added the comment.

If the comment seems misleading, I can just remove it.

Reply:
ahh yeah. `flags=defaults` just means set the flags as the defaults are, without touhing all other values
consider rewording it, not sure

Reply:
I've now at least removed the second half of the comment.